src/System.Device.Gpio.Tests/README.md

```diff
@@ -0,0 +1,39 @@
+# How to run tests on the Raspberry Pi
```

Comment:
Overall comment:
- I'm currently in the process of changing these tests and the new version will include diagram what needs to be connected: https://github.com/dotnet/iot/pull/758
- We can merge these but expect I'll modify these to adjust for changes (I might add something to make it easier to run when people have incomplete setup)

Reply:
I did this as a preparation for another commit, so I guess this can be merged and used as a starting point. 
Since not everybody will have all the hardware to run all the tests (i.e. the ADC converter), selecting tests will be important. Note also that the current input/output tests have no timeout defined, so they will block if the pins are not connected.

Reply:
> Note also that the current input/output tests have no timeout defined, so they will block if the pins are not connected.

For this we were relying on our testing environment which limits the time a test will be executed, and kill it (and mark it as failed obviously) if it is just hanging.

Reply:
Relying on an external process to terminate the tests is not really a good design for errors that you expect to happen, I think. Especially since here it's easy to be fixed, as there are already CancellationTokens in place. I can have a look into that.

Reply:
That should be fixed on a separate PR, I think.